worker/diskmanager/diskmanager.go

```diff
@@ -64,7 +64,10 @@ func doWork(listf ListBlockDevicesFunc, b BlockDeviceSetter, old *[]storage.Bloc
 		logger.Tracef("no changes to block devices detected")
 		return nil
 	}
-	logger.Infof("block devices changed: %#v", blockDevices)
+	logger.Infof("block devices changed")
+	if logger.IsTraceEnabled() {
```

Comment:
**todo:** I don't think that it's useful to check against is TraceEnabled, since there is no compute required to get the log, and the trace function will do this check very early, before any argument processing.

https://github.com/juju/loggo/blob/ebdeb290b96116b911ea93cbe62107921610346e/logger.go#L293C1-L297C2

https://github.com/juju/loggo/blob/ebdeb290b96116b911ea93cbe62107921610346e/logger.go#L206